cypress/e2e/addPerson.feature

```diff
@@ -1,8 +1,9 @@
+@personal-details
```

Comment:
Confusingly and wrongly enough.

I believe the `add person` functionality is actually part of the tenure MFE [[link](https://github.com/LBHackney-IT/mtfh-frontend-tenure/blob/3c8f485d9075d3ed645dc4f50b6181cf52e3b35d/src/views/tenure-steppers-view/tenure-steppers-view.tsx#L29)]. It's pretty evident when you look into the URL of the create person page.